<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSqlConnectionDialogUi</class>
 <widget class="QDialog" name="QSqlConnectionDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect...</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="connectionGroupBox">
     <property name="title">
      <string>Строка подключения</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin" stdset="0">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboDriver"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="username">
        <property name="text">
         <string>Пользователь</string>
        </property>
        <property name="buddy">
         <cstring>editUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="driver">
        <property name="text">
         <string>СУБД</string>
        </property>
        <property name="buddy">
         <cstring>comboDriver</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editDatabase"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="specialValueText">
         <string> По умолчанию</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="databaseName">
        <property name="text">
         <string>База данных</string>
        </property>
        <property name="buddy">
         <cstring>editDatabase</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editPassword">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editUsername"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editHostname"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="hostname">
        <property name="text">
         <string>Хост</string>
        </property>
        <property name="buddy">
         <cstring>editHostname</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="port">
        <property name="text">
         <string>Порт</string>
        </property>
        <property name="buddy">
         <cstring>portSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="password">
        <property name="text">
         <string>Пароль</string>
        </property>
        <property name="buddy">
         <cstring>editPassword</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="memoryDatabase">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="databaseCheckBox">
       <property name="text">
        <string>Использовать предопределенную базу данных в памяти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ок</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboDriver</tabstop>
  <tabstop>editDatabase</tabstop>
  <tabstop>editUsername</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editHostname</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>databaseCheckBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
